examples/standalone/runfile/dynamics.py

```diff
@@ -182,27 +179,21 @@ def collect_data_and_write_to_file(
         fv3core.set_backend(args.backend)
         fv3core.set_rebuild(False)
         fv3core.set_validate_args(False)
-        global_config.set_do_halo_exchange(not args.disable_halo_exchange)
 
         spec.set_namelist(args.data_dir + "/input.nml")
 
-        experiment_name = yaml.safe_load(
-            open(
-                args.data_dir + "/input.yml",
-                "r",
-            )
-        )["experiment_name"]
+        experiment_name = os.path.basename(os.path.normpath(args.data_dir))
```

Comment:
I see this being shorter, but is it not significantly less safe to rely on the folder name to be something specific compared to reading the `yml` file?

Reply:
The point wasn't that it's shorter, it's that we actually do not have this yaml file in the data download. I have no idea where this yaml file comes from. This name is also the only thing we use from that yaml file, so we should probably just be passing the experiment name as an argument instead of a path to a yaml file with an experiment name. I     
 can change it to do that instead, but this involves changing a jenkins plan that is not tested on PRs so it's more risky, and debugging is a many-hour turnaround.

Reply:
I will restore the use of `input.yml` since we'll probably use it in the long term. This will happen by moving the "move experiment.yml to input.yml" command into the make target instead of in the jenkins scripts. In the long term, we want to rename the experiment yml's to `input.yml`. I'll make a JIRA task for this.

Reply:
Since you have the latter part of this plan implemented and in a PR, instead I will revert the code here and we can use your full solution. This was changed only to fix the make target working locally, so it should be OK to leave that not-working (requiring a manual `mv`) for another few days/week.